[DYCLOUD]2024-11-08 09:00:54.731	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 09:21:59.762	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:21:59.763	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/container.go:39	获取容器列表失败:	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:21:59.763	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:21:59.764	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/image.go:33	获取镜像列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:21:59.764	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:21:59.764	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/volume.go:33	获取存储卷列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:21:59.765	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:21:59.765	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/system.go:29	获取系统信息失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:21:59.766	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:21:59.766	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/network.go:33	获取网络列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:25:48.962	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/service/configCenter/serviceIntegration.go:197	登录仓库失败:http://http://192.168.31.41:82访问异常:Get "http://http//192.168.31.41:82": EOF
[DYCLOUD]2024-11-08 09:26:09.932	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/service/configCenter/serviceIntegration.go:197	登录仓库失败:http://http://192.168.31.41:82访问异常:Get "http://http//192.168.31.41:82": EOF
[DYCLOUD]2024-11-08 09:26:11.826	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/service/configCenter/serviceIntegration.go:197	登录仓库失败:http://http://192.168.31.41:82访问异常:Get "http://http//192.168.31.41:82": EOF
[DYCLOUD]2024-11-08 09:31:09.619	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/kubernetes/deployments.go:42	获取失败: record not found
[DYCLOUD]2024-11-08 09:31:09.619	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/service/kubernetes/node/k8s_node.go:44	record not found
[DYCLOUD]2024-11-08 09:32:13.783	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/kubernetes/configManager/configmap/k8s_configmap.go:68	获取失败!	{"error": "resource name may not be empty"}
[DYCLOUD]2024-11-08 09:32:26.032	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/kubernetes/configManager/configmap/k8s_configmap.go:68	获取失败!	{"error": "resource name may not be empty"}
[DYCLOUD]2024-11-08 09:33:19.984	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:33:19.984	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:33:19.984	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/container.go:39	获取容器列表失败:	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:33:19.984	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:33:19.984	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/system.go:29	获取系统信息失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:33:19.985	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/volume.go:33	获取存储卷列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:33:19.985	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:33:19.985	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/network.go:33	获取网络列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:33:19.986	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:33:19.986	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/image.go:33	获取镜像列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:35:40.013	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:35:40.014	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/system.go:29	获取系统信息失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:35:40.014	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:35:40.014	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/volume.go:33	获取存储卷列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:35:41.058	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:35:41.058	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/network.go:33	获取网络列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:35:42.018	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:35:42.019	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/image.go:33	获取镜像列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:35:42.027	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:35:42.027	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:35:42.029	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/system.go:29	获取系统信息失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:35:42.030	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/container.go:39	获取容器列表失败:	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:35:42.033	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:35:42.034	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/volume.go:33	获取存储卷列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 09:35:42.048	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/utils/docker/docker/client.go:35	record not found
[DYCLOUD]2024-11-08 09:35:42.048	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/api/v1/docker/network.go:33	获取网络列表失败!	{"error": "host is not found"}
[DYCLOUD]2024-11-08 10:30:27.118	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 10:30:59.396	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/initialize/redis.go:31	redis connect ping failed, err:	{"name": "", "error": "MISCONF Redis is configured to save RDB snapshots, but it's currently unable to persist to disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error."}
[DYCLOUD]2024-11-08 10:33:27.536	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 10:35:12.488	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 11:13:03.594	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 13:58:02.783	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 14:03:46.861	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 14:05:29.667	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 14:06:36.738	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 14:14:39.088	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
[DYCLOUD]2024-11-08 14:17:37.680	[31merror[0m	/Users/dujie/Desktop/cmdb/持续更新/cmdb/server/core/server.go:41	accept tcp [::]:8888: use of closed network connection
